products/sqlite.md

```diff
@@ -0,0 +1,333 @@
+---
+title: SQLite
+category: db
+iconSlug: sqlite
+permalink: /sqlite
+releasePolicyLink: https://www.sqlite.org/news.html
```

Comment:
Is there a page on the SQLite explaining their release policy ? If yes this link should be used, if no using `https://www.sqlite.org/` would be best.

Reply:
Fixed :+1:

Reply:
I still see the same link.

Also found this one which would be more appropriate : https://sqlite.org/com/tech.html. The extended support column could be added too.

Reply:
Finally used https://www.sqlite.org/support.html.